src/System.Private.CoreLib/src/System/Threading/Tasks/ThreadPoolTaskScheduler.cs

```diff
@@ -94,7 +95,7 @@ protected override IEnumerable<Task> GetScheduledTasks()
             return FilterTasksFromWorkItems(ThreadPool.GetQueuedWorkItems());
         }
 
-        private IEnumerable<Task> FilterTasksFromWorkItems(IEnumerable<IThreadPoolWorkItem> tpwItems)
+        private IEnumerable<Task> FilterTasksFromWorkItems(IEnumerable<object> tpwItems)
```

Comment:
The loop below should be changed to handle `object` instead of the interface

Reply:
Yup, will fix.  Good catch.

Reply:
Fixed